docs/model-dev-guide/api-guides/apis-howto/api-keras-ug.rst

```diff
@@ -7,129 +7,155 @@
 .. meta::
    :description: Learn how to use the Keras API to train a Keras model. This user guide walks you through loading your data, defining the model, customizing how the model.fit function is called, checkpointing, and callbacks.
 
-In this guide, you'll learn how to use the Keras API.
+In this guide, you'll learn how to use the Determined's keras.DeterminedCallback while training your
+keras model.
 
 +---------------------------------------------------------------------+
 | Visit the API reference                                             |
 +=====================================================================+
 | :ref:`keras-reference`                                              |
 +---------------------------------------------------------------------+
 
-This document guides you through training a Keras model in Determined. You need to implement a trial
-class that inherits :class:`~determined.keras.TFKerasTrial` and specify it as the entrypoint in the
-:ref:`experiment-configuration`.
+This document guides you through training a Keras model in Determined.  You will need to update your
+``model.fit()`` call to include a :class:`~determined.keras.DeterminedCallback` and submit it to
+a Determined cluster.
 
-To learn about this API, you can start by reading the trial definitions in the `Iris categorization
-example
+To learn about this API, you can start by reading the ``train.py`` script in the `Iris
+categorization example
 <https://github.com/determined-ai/determined-examples/tree/main/computer_vision/iris_tf_keras>`__.
 
-***********
- Load Data
-***********
+**********************
+ Configure Entrypoint
+**********************
 
-.. note::
+Determined requires you to launch training jobs by submitting them with an
+:ref:`experiment-configuration` that tells the Determined master how to start your container.  For
```

Comment:
```suggestion
:ref:`experiment-configuration` which tells the Determined master how to start your container. For
```

Reply:
question: don't you need a comma before "which", but not before "that"?